sporks.json

```diff
@@ -3,6 +3,34 @@
     {
       "name": "mainnet",
       "sporks": [
+        {
+          "id": 15,
+          "name": "Mainnet-15",
+          "spork-time": "2021-12-09T16:00:00Z",
+          "root-height": "21291692",
+          "root-parent-id": "8a28ddc6653a8991435dfbc95103d6be1a3b653cda3d664681215ee93112a203",
+          "root-state-commit": "84d9b325d3f48f7f075eea0db6c246cb5adc759a2dd8561e461477a7ca1f7f22",
+          "git-commit-hash": "bfde54ae3324db5d18ebeaa22c70b5574a114f2e",
+          "state-artefacts" : {
+            "gcp": {
+              "root-checkpoint-file": "https://storage.googleapis.com/flow-genesis-bootstrap/mainnet-15-execution/public-root-information/root.checkpoint",
+              "root-protocol-state-snapshot": "https://storage.googleapis.com/flow-genesis-bootstrap/mainnet-15-execution/public-root-information/root-protocol-state-snapshot.json",
+              "node-info": "https://storage.googleapis.com/flow-genesis-bootstrap/mainnet-15-execution/public-root-information/node-infos.pub.json",
+              "execution-state-wal-chunks": "https://storage.googleapis.com/flow_public_mainnet15_execution_state"
+            },
+            "s3": {
```

Comment:
there is an AWS s3 bucket now `flow-public-mainnet15-execution-state` (same name as GCP but dashes instead of underscores)

Reply:
for the execution state wal chunks (not for anything else)

Reply:
Oh great! Let me check on the contents, thanks

Reply:
@vishalchangrani added the EXE state bucket for mainnet15, but the other buckets aren't yet spun up and populated